src/lib/components/quiz/quiz.svelte

```diff
@@ -0,0 +1,249 @@
+<script lang="ts">
+  import type { QuizData, TeamMatch } from '$lib/constants/quiz';
+  import LeftArrow from '$lib/components/icons/left-arrow.svelte';
+  import RightArrow from '$lib/components/icons/right-arrow.svelte';
+  import MoreInfo from './more-info.svelte';
+
+  export let data: QuizData;
+
+  $: index = 0;
```

Comment:
Since we are setting the variable `index` to `0`, we do not need to set this variable with the _reactive_ syntax.
```suggestion
  let index = 0;
```

Reply:
done